src/js_test/test_ir_remote.js

```diff
@@ -20,17 +20,68 @@ var zmq = require('zmq')
 var configSocket = zmq.socket('push')
 configSocket.connect('tcp://' + creator_ip + ':' + creator_lirc_base_port /* config */)
 
-function sendIrCommand() {
+var http = require('http');
+var fs = require('fs');
+
+/**
+ * LIRC remote database on admobilize assets.
+ * Orginal source on: https://sourceforge.net/p/lirc-remotes/code/ci/master/tree/
+ */
+
+// Example remote device (SONY control RM-AAU014)
+var remote_url="http://assets.admobilize.com/lirc-remotes/sony/RM-AAU014.lircd.conf"
```

Comment:
Is it OK to download from here all the time? Maybe we should copy the file and download from one of our servers. Just wondering.

We can leave this as it is for now.